packages/arb-bridge-eth/contracts/rollup/RollupCore.sol

```diff
@@ -262,6 +251,7 @@ contract RollupCore is IRollupCore {
         bytes32 afterSendAcc = RollupLib.feedAccumulator(sendsData, sendLengths, beforeSendAcc);
 
         INode node = getNode(nodeNum);
+        // Replay protect:
```

Comment:
This is more of a pre-image review to get the accumulators and counts out of the node than replay protection